sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_access_control_client.py

```diff
@@ -38,8 +38,9 @@ class KeyVaultAccessControlClient(KeyVaultClientBase):
     # pylint:disable=protected-access
 
     @distributed_trace
-    def create_role_assignment(self, scope, definition_id, principal_id, **kwargs):
-        # type: (Union[str, KeyVaultRoleScope], str, str, **Any) -> KeyVaultRoleAssignment
+    def create_role_assignment(
+        self, scope: "Union[str, KeyVaultRoleScope]", definition_id: str, principal_id: str, **kwargs: "Any"
```

Comment:
nit: Why not get rid of `TYPE_CHECKING`? Then don't need the quotes for all type hints.

Reply:
You can remove hints for `kwargs` which is the comment I got: https://github.com/Azure/azure-sdk-for-python/pull/28095#discussion_r1060868828

Reply:
> nit: Why not get rid of TYPE_CHECKING?

That's something I'd like to do eventually, but given that there are already a lot of changes in this PR, I was thinking of saving that for another time.